main.py

```diff
@@ -0,0 +1,18 @@
+import pathlib
+
+import click
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command()
+@click.argument("settings", click.Path)
+def evaluate(settings: pathlib.Path):
+    ...
```

Comment:
would this evaluate be hit by github CLI to run benchmarks against a path to the github cli?